src/middleware.ts

```diff
@@ -368,22 +368,23 @@ export const persist =
             )
           } else {
             stateFromStorageInSync = deserializedStorageValue.state
```

Comment:
```suggestion
            return deserializedStorageValue.state
```
Now, it seems we can simplify this and remove L375 if-statement.